### ISSUE TITLE
static build fails in tests

### ISSUE THREAD

**Issue by marksantcroos** [open-mpi/ompi#1112]
```
$ ../configure --enable-orterun-prefix-by-default --enable-static --disable-shared --prefix=$MEMBERWORK/csc168/openmpi/installed/STATIC
$ make install
```
```
Making install in monitoring
make[2]: Entering directory `/autofs/nccs-svm1_home1/marksant1/openmpi/src/ompi/build/test/monitoring'
  CC       monitoring_prof.lo
  CCLD     monitoring_prof.la
libtool: link: can not build a shared library
libtool: link: See the libtool documentation for more information.
libtool: link: Fatal configuration error.
make[2]: *** [monitoring_prof.la] Error 1
make[2]: Leaving directory `/autofs/nccs-svm1_home1/marksant1/openmpi/src/ompi/build/test/monitoring'
make[1]: *** [install-recursive] Error 1
make[1]: Leaving directory `/autofs/nccs-svm1_home1/marksant1/openmpi/src/ompi/build/test'
make: *** [install-recursive] Error 1
```

**Comment by hppritcha** [comment]
I see this same error on a vanilla x86_64 whitebox running Open Suse 13.1.  This is not a cray specific problem.

**Comment by hppritcha** [comment]
I'm testing a simple fix.

**Comment by marksantcroos** [comment]
Renamed appropriately :-)

**Comment by ggouaillardet** [comment]
@hppritcha same thing on a RHEL7 like box
@marksantcroos i fixed that in open-mpi/ompi@c415ecf39e91fe87705cbd4f18265ba91fe6a30c

**Comment by marksantcroos** [comment]
Thanks!
